test/test_cases/run_tests.sh

```diff
@@ -7,17 +7,17 @@ TEST_NAME_PREFIX=${1:-TEST_ID_}
 # Keep environment variables for debugging
 printenv > printenv.debug
 
-export TEST_CASES_DIR="$PWD/../config/k8s"
+export ENV_DIR="$PWD/../../common/config/k8s"
```

Comment:
what does `ENV_DIR` stand for? is it a directory that holds the env? seems that `CONFIG_DIR` will fit better

Reply:
Updated (:

Reply:
fixed!